src/Wrapper.ts

```diff
@@ -150,27 +152,23 @@ export default function<T, V>(
       this.setValidations(validations, required);
 
       // Pass a function instead?
-      formsy.attachToForm(this);
+      attachToForm(this);
```

Comment:
Is it considered better to have your context not be wrapped in a single namespace like `formsy` as it was before?

Reply:
The way we subscribe the the context is with the static property ContextType. Subscribe to multiple contexts with a static property are deprecated.